dev/codesniffer/ruleset.xml

```diff
@@ -3,11 +3,16 @@
 <ruleset name="Dolibarr">
 	<description>Dolibarr coding standard.</description>
 
-	<exclude-pattern>*/conf.php</exclude-pattern>
-	<exclude-pattern>*/includes/*</exclude-pattern>
-	<exclude-pattern>*/documents/*</exclude-pattern>
-	<exclude-pattern>*/dev/vagrant/*</exclude-pattern>
-	<exclude-pattern>*/custom/*</exclude-pattern>
+	<exclude-pattern>build/html</exclude-pattern>
+	<!--
```

Comment:
Oh crap. I figured it out. I have a documents folder in my path !
Pull request using relative path instead incoming.